newIDE/app/src/GameDashboard/LeaderboardAdmin/LeaderboardAppearanceDialog.js

```diff
@@ -175,6 +216,36 @@ function LeaderboardAppearanceDialog({
             />,
           ]}
         >
+          <Text size="title">
+            <Trans>Table settings</Trans>
+          </Text>
+          <Line>
+            <TextField
+              fullWidth
+              type="number"
+              floatingLabelText={<Trans>Number of entries to display</Trans>}
+              value={
+                isNaN(defaultDisplayedEntriesNumber)
+                  ? ''
```

Comment:
if it's a `type="number"` this should probably be a number too no?
(This check may not be needed as you're doing it above too)

Reply:
It's necessary to allow users to delete the whole number and type a new, otherwise, when deleting the last digit, it sets the value to 0